demo/data/rags_compare.csv

```diff
@@ -0,0 +1,10 @@
+question
```

Comment:
Can we automatically generate questions now ?

or we can just upload questions in csv format

Reply:
We can only upload questions in csv format now, and the current version is using the questions from huggingface datasets. Uploading our questions need different params for the class ‘retriever_model_prompt' to init.